samples/nrf9160/modem_shell/README.rst

```diff
@@ -343,6 +373,10 @@ Examples
      sock rai -i 0 --rai_last
      sock send -i 0 -d testing
 
+  When both 3GPP Release 13 Control Plane (CP) RAI and 3GPP Release 14 Access Stratum (AS) RAI are enabled,
+  which can be the case for NB-IOT, then both are signalled.
+  It depends on the network configuration and prioritization, which RAI takes effect.
```

Comment:
```suggestion
  When both 3GPP Release 13 Control Plane (CP) RAI and 3GPP Release 14 Access Stratum (AS) RAI are enabled,
  which can be the case for NB-IOT, both are signalled.
  Which RAI takes effect depends on the network configuration and prioritization.
```
Have we introduced RAI before? If not, should we want to open the abbreviation?

Reply:
Not in this document page but elsewhere yes. I will add this.